bigtop-packages/src/common/alluxio/install_alluxio.sh

```diff
@@ -65,12 +66,15 @@ while true ; do
         --bin-dir)
         BIN_DIR=$2 ; shift 2
         ;;
-        --var-dir)
-        VAR_DIR=$2 ; shift 2
+        --var-alluxio-dir)
+        VAR_ALLUXIO_DIR=$2 ; shift 2
```

Comment:
Is this crucial? If we really need this, [rules](https://github.com/apache/bigtop/blob/0f67454a7b71f54b6098ef44cb459e656853a0d7/bigtop-packages/src/deb/alluxio/rules#L39) should be updated too for DEB packaging.

Reply:
Thank you for the reminder. I have checked the Alluxio cluster, configuration, and Ambari code. Indeed, the /var/lib/alluxio directory is not being utilized. I have removed this directory from the packaged code.